omod/src/main/webapp/resources/scripts/custom/deletePatient.js

```diff
@@ -0,0 +1,46 @@
+var delPatient = delPatient || {};
+
+delPatient.deletePatientCreationDialog = null;
+
+/**
+ * Functions used to Delete a Patient (uses DeletePatientFragmentController)
+ */
+
+delPatient.returnUrl = "/coreapps/findpatient/findPatient.page?app=coreapps.findPatient";
+
+delPatient.createDeletePatientCreationDialog = function() {
+    delPatient.deletePatientCreationDialog = emr.setupConfirmationDialog({
+        selector: '#delete-patient-creation-dialog',
+        actions: {
+            confirm: function() {
+                emr.getFragmentActionWithCallback('coreapps', 'deletePatient', 'deletePatientEntry',
+                    { patientId: delPatient.patientId},
```

Comment:
I need to pass in the message from the textbox `retire-reason` here. How do I do it?